src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -22,8 +26,17 @@ internal class SNIProxy
         private static readonly char BackSlashSeparator = '\\';
```

Comment:
same for the other two

---
In reply to: [100624800](https://github.com/dotnet/corefx/pull/15742#discussion_r100624800) [](ancestors = 100624800)